### DISCUSSION TITLE
AARCH64 teamtalk.

### DISCUSSION THREAD

**Discussion by CatsAreCool710** [BearWare/TeamTalk5#1851] [General]
Is there an AARCH64 binary for the client? A lot of new arm devices including linux on arm platforms use AARCH64

**Comment by bear101** [comment]
No, there isn't. You have to build it yourself. I don't have access to an ARM computer except Raspberry Pi.

**Comment by CatsAreCool710** [comment]
Would I need the SDK to build it?

**Reply by bear101** [reply]
No. Clone GitHub project and try to build.

**Comment by CatsAreCool710** [comment]
I'm running asahi linux which is based on archlinuxarm. Should I try this in a docker, or what's the best way to build?

**Reply by bear101** [reply]
Up to you. I don't know how to do it.
